etc/insights-client.spec

```diff
@@ -136,8 +135,11 @@ if ! [ -d "/var/lib/insights" ]; then
 mkdir -m 644 /var/lib/insights
 fi
 
-# symlink the motd file
-ln -snf /etc/insights-client/insights-client.motd /etc/motd.d/insights-client
+#Link motd into place. Only do so if motd file doesn't exist
```

Comment:
What happened to checking for new install vs. upgrade? We still only want to do this on fresh installs, AFAIUI.

Reply:
No we do want it to appear on upgrades to 8.1 or to 8 z-stream ... just not *reappear* on upgrades if the system already uses insights.

Reply:
Hence the logic of, if the symlink exists, do nothing. And insights-client always keeps a symlink there ... albeit to ```/dev/null``` once insights-client has been used.

Reply:
Nevermind, you said above that they *should* appear on upgrades.